### DISCUSSION TITLE
Release v3.21.0?

### DISCUSSION THREAD

**Discussion by dennisguse** [OpenTracksApp/OpenTracks#998] [General]
@rgmf is there anything blocking us from releasing v3.21.0?
We accumulated quite some changes :)

**Comment by rgmf** [comment]
@dennisguse nothing blocked. Go ahead.

**Reply by dennisguse** [reply]
Good morning by the way :sunglasses:

**Comment by rgmf** [comment]
🤣 good morning

**Reply by dennisguse** [reply]
:sunrise:

**Comment by dennisguse** [comment]
Released.

PS/ the changelog for v3.21.0 is really long (also with cool stuff).
:sunglasses: 

@vlmendz you did a great job, digging in the recording code.
Appreciated.

**Reply by vlmendz** [reply]
Thanks :)
